### PR TITLE
[Fix] Configuration files merge sequence

### PR DESCRIPTION
**Pull request by zhenik**

### CODE REVIEW

File: ansible/bootstrap.yml [hunk 1]
```diff
@@ -107,6 +107,32 @@
...
+        status_code: 200
+        headers:
+          X-Consul-Token: "{{ token }}"
+        body_format: json
+      register: services_payload_2
+      until: services_payload_2.status == 200
+      retries: 5
+      delay: 1
+      when: lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'deny'
+      failed_when: services_payload_2.json["consul"] is not defined
```
Comment (pdmthorsrud) [inline ansible/bootstrap.yml:134]: I feel like we are getting to a point where these tests could be separated into its own playbook. WDYT?

Comment (zhenik) [reply ansible/bootstrap.yml:134]: Yeah. I agree. Might be useful to have a separate folder under ansible

Comment (pdmthorsrud) [reply ansible/bootstrap.yml:134]: Yeah, definitely add a folder, because there's a lot of playbooks under `ansible/ ` right now.

Comment (pdmthorsrud) [reply ansible/bootstrap.yml:134]: If you feel like it we could just make an issue for this, and merge this PR as is first!

Comment (zhenik) [reply ansible/bootstrap.yml]: done it here 3b8aa51

**Review by pdmthorsrud — APPROVED** [review]

**Review by pdmthorsrud — DISMISSED** [review]

File: ansible/tests/consul_acl_test.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    status_code: 200
+    headers:
+      X-Consul-Token: "master"
+  register: result
+  until: result.status == 200
+  retries: 5
+  delay: 1
+  when: lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'deny'
+
+- name: default_policy=deny - list services without token, does not show servie consul
```
Comment (claesgill) [inline ansible/tests/consul_acl_test.yml:23]:
```suggestion
- name: default_policy=deny - list services without token, does not show service in consul
```

**Review by claesgill — APPROVED** [review]
LGTM 🚀

**Review by pdmthorsrud — APPROVED** [review]
LGTM

**Review by fredrikhgrelland — APPROVED** [review]
LGTM

**Pull request merged**